redis/parsers/commands.py

```diff
@@ -1,6 +1,11 @@
+from typing import TYPE_CHECKING, Any, Dict, Optional, Tuple, Union
```

Comment:
This file & class were always misnamed. It should ideally have "cluster" somewhere in there.